integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/webhooks/UrlPusher.java

```diff
@@ -26,7 +26,7 @@ void push(String uri, String jsonPayload) {
             ResponseEntity<String> responseEntity = restTemplate.exchange(uri, HttpMethod.POST, new HttpEntity<>(jsonPayload, headers), String.class);
             log.debug("Response was {} {}", responseEntity.getStatusCode().value(), responseEntity.getStatusCode().getReasonPhrase());
         } catch (ResourceAccessException e) {
-            log.warn("Webhook push failed for {}", uri);
+            log.warn("Webhook push failed for {}: {}", uri, e);
```

Comment:
Dette gjør vel kun en toString() av exceptionen? Sjekk parameterene til log.warn for å logge med stacktrace..